### PR TITLE
Add jnv

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pavelzw** [comment]
Issue looks like https://github.com/rust-lang/cc-rs/issues/958

File: recipes/jnv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  license: MIT
+  license_family: MIT
+  license_file:
+    - LICENSE
+    - THIRDPARTY_LICENSES.yaml
+  summary: JSON navigator and interactive filter leveraging jq
+
+extra:
+  recipe-maintainers:
+    - pavelzw
```
Comment (moritzwilksch) [inline recipes/jnv/meta.yaml:44]:
```suggestion
    - pavelzw
    - moritzwilksch
```

**Review by moritzwilksch — CHANGES_REQUESTED** [review]
🙃

**Comment by pavelzw** [comment]
```
  checking host system type... 
  --- stderr
  configure.ac:11: installing 'config/ar-lib'
  configure.ac:8: installing 'config/compile'
  configure.ac:19: installing 'config/config.guess'
  configure.ac:19: installing 'config/config.sub'
  configure.ac:9: installing 'config/install-sh'
  configure.ac:9: installing 'config/missing'
  Makefile.am: installing 'config/depcomp'
  configure.ac: installing 'config/ylwrap'
  parallel-tests: installing 'config/test-driver'
  Invalid configuration `/home/conda/staged-recipes/build_artifacts/jnv_1712488275842/_build_env/bin/x86_64-conda-linux-gnu': more than four components
  configure: error: /bin/sh /home/conda/staged-recipes/build_artifacts/jnv_1712488275842/work/target/x86_64-unknown-linux-gnu/release/build/j9-sys-b50a7ed31c9b8fd9/out/jq_build/config/config.sub /home/conda/staged-recipes/build_artifacts/jnv_1712488275842/_build_env/bin/x86_64-conda-linux-gnu failed
  thread 'main' panicked at /home/conda/staged-recipes/build_artifacts/jnv_1712488275842/_build_env/.cargo/registry/src/index.crates.io-6f17d22bba15001f/autotools-0.2.6/src/lib.rs:781:5:
```

**Comment by pavelzw** [comment]
@conda-forge/help-rust this is ready for review

File: recipes/jnv/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export CARGO_BUILD_RUSTFLAGS="${CARGO_BUILD_RUSTFLAGS} -L$PREFIX/lib"
+
+cargo-bundle-licenses \
+    --format yaml \
+    --output THIRDPARTY_LICENSES.yaml
+
+# TODO: add --locked
+cargo install --root "$PREFIX" --path .
```
Comment (xhochy) [inline recipes/jnv/build.sh:10]:
```suggestion
cargo install --no-track --root "$PREFIX" --path .
```

File: recipes/jnv/bld.bat [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cargo-bundle-licenses ^
+    --format yaml ^
+    --output THIRDPARTY_LICENSES.yaml || goto :error
+
+REM TODO: add --locked
+cargo install --root "%PREFIX%" --path . || goto :error
```
Comment (xhochy) [inline recipes/jnv/bld.bat:6]:
```suggestion
cargo install --no-track --root "%PREFIX%" --path . || goto :error
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Pull request merged**